curator-framework/src/main/java/org/apache/curator/framework/api/DeleteBuilder.java

```diff
@@ -18,6 +18,6 @@
  */
 package org.apache.curator.framework.api;
 
-public interface DeleteBuilder extends Guaranteeable, ChildrenDeletable
+public interface DeleteBuilder extends GuaranteeableDelete, ChildrenDeletable
```

Comment:
A nit, but the naming convention is actually: <Verb1>..<VerbN>able. So it would be GuaranteeDeletable.